pkg/utils/utils.go

```diff
@@ -53,3 +56,9 @@ func GetFilesWithSuffix(baseDir string, suffixes ...string) ([]string, error) {
 	}
 	return files, nil
 }
+
+func GetNetworkAttachmentDefLink(netAttDef *netattdefv1.NetworkAttachmentDefinition) string {
+	link := fmt.Sprintf("%s/namespaces/%s/%s/%s",
```

Comment:
instead of link := you can just return
another option is to define in the signature
func GetNetworkAttachmentDefLink(netAttDef *netattdefv1.NetworkAttachmentDefinition) (link string)
{
link = fmt.Sprintf("%s/namespaces ...
}

Reply:
@moshe010 could you please add more details on  this option?

> another option is to define in the signature

Reply:
> @moshe010 could you please add more details on this option?
> 
> > another option is to define in the signature
sorry you still need to return. It called Named return values
https://tour.golang.org/basics/7

Reply:
Thank you, I'll update a PR